### DISCUSSION TITLE
Bad gateway

### DISCUSSION THREAD

**Discussion by wojtekh** [easypanel-io/community#83] [Q&A]
i am getting a bad gateway error after  app install and using custom domain
Domain is not proxied, i am using cloudflare for DNS
Any ideas are appreciated

**Comment by deadcoder0904** [comment]
@wojtekh did you solve it? i am facing the same issue.

**Reply by mlc-studio** [reply]
Same today with a new vps

**Comment by govtech42** [comment]
I was having the same problem. I couldn't solve it.

That's why I changed my DNS to Vultr. After the change, everything worked properly.

**Reply by natanaelece** [reply]
Did you make the change in the vm?

**Comment by natanaelece** [comment]
I have the same problem.
I installed EasyPanel in a container on Proxmox, but when accessing the domain it gives a bad gateway error. Does anyone know how to solve this?
